### ISSUE TITLE
Resolve focus issue when inserting block

### ISSUE THREAD

**Issue by ronalfy** [DLXPlugins/quotes-dlx#3]
The cursor should be at the beginning of the quote text box when first inserted. a11y

End of quote when when the appearance tab is selected. a11y

End of quote when block is focused. a11y
